packages/ckeditor5-utils/src/translation-service.ts

```diff
@@ -9,16 +9,13 @@
  * @module utils/translation-service
  */
 
+import type { Translations, Editor } from '@ckeditor/ckeditor5-core';
```

Comment:
I'm worried about circular dependency. Do we really want to make `utils` dependent on `core`? I know it is `devDependencies`, but still.
@filipsobol, @niegowski ?